packages/arb-avm-cpp/avm/include/avm/machinestate/datastack.hpp

```diff
@@ -84,7 +84,7 @@ class Datastack {
     }
 
     std::pair<HashPreImage, std::vector<unsigned char>> marshalForProof(
-        const std::vector<bool>& stackInfo,
+        const std::vector<MarshalLevel>& stackInfo,
```

Comment:
nice

Reply:
Haha yeah. That let me get rid of the marshal stub functions entirely which was nice